database_backend/backendQueries.js

```diff
@@ -0,0 +1,106 @@
+console.log("server is starting");
+var express = require("express");
+var app = express();
+var fs = require("fs");
+var body = require("body-parser");
+var cors = require('cors');
+app.use(sors());
+app.use(body.urlencoded({ extended: false }));
+app.listen(3000);
+app.use(body.json());
+
+const cors = require("cors");
+app.use(cors());
+
+const Pool = require("pg").Pool;
+const pool = new Pool({
+  user: "postgres",
+  host: "localhost",
+  database: "vikasmishra",
+  password: "Source@123",
+
+  port: 5432
+});
+//FETCH DATA API
+app.get("/bussinessLogic", getdata);
+function getdata(request, response) {
+  console.log("ll");
+  pool.query("SELECT * FROM employee ORDER BY id ASC", (error, results) => {
+    if (error) {
+      throw error;
+    }
+    response.status(200).json(results.rows);
+  });
+}
+//DELETE DATA API
+app.delete("/maintype/:id", (request, response) => {
```

Comment:
api urls should be meaningful names. like when you are computing on employees. so it should be /employee/add,/employee/update